docs/tutorials/sml-classification.ipynb

```diff
@@ -0,0 +1,818 @@
+    "\n",
+    "- More learners expand the hypothesis space, allowing the model to capture more subtle patterns, which can lift F1 and recall.\n",
+    "- However, wider circuits mean higher **QPU load** (more qubits to map and deeper transpilation), increasing runtime and cost.\n",
+    "- The improvement in quality typically follows a *diminishing-returns curve*: early gains are significant, but plateaus as the ensemble saturates.\n",
+    "\n",
+    "This experiment makes the quality–cost trade-off visible — more width improves results, but at the expense of resource efficiency."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e5fe64b0-2713-4b65-b768-10fa5d8dbf79",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Problem scale and regularization\n",
+    "NUM_LEARNERS = 120\n",
+    "REGULARIZATION = 7"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "18a22fed-41c2-4407-88d4-02fbd32f3320",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ----- Quantum-enhanced ensemble on IBM hardware -----\n",
+    "print(\"\\n-- Submitting quantum-enhanced ensemble job --\")\n",
+    "job_2 = singularity.run(\n",
+    "    action=\"create_fit_predict\",\n",
+    "    name=\"grid_stability_qeec\",\n",
+    "    quantum_classifier=\"QuantumEnhancedEnsembleClassifier\",\n",
+    "    num_learners=NUM_LEARNERS,\n",
+    "    regularization=REGULARIZATION,\n",
+    "    optimizer_options=optimizer_options,  # from Step 2\n",
+    "    backend_name=backend,  # least-busy compatible backend\n",
+    "    instance=IBM_INSTANCE_QUANTUM,\n",
+    "    random_state=RANDOM_STATE,\n",
+    "    X_train=X_train_bal,\n",
+    "    y_train=y_train_bal,\n",
+    "    X_test=X_test,\n",
+    "    fit_params={\"validation_data\": (X_val, y_val)},\n",
+    "    options={\"save\": False},\n",
+    ")\n",
+    "status_2 = job_2.status()\n",
+    "result_2 = job_2.result()\n",
+    "print(\"\\nQuantum job status:\", status_2)\n",
+    "print(\"Action status:\", result_2.get(\"status\"))\n",
+    "print(\"Action message:\", result_2.get(\"message\"))\n",
+    "print(\"QPU Time:\", result_2.get(\"metadata\"))\n",
+    "qeec_pred_job_2 = np.array(result_2[\"data\"][\"predictions\"])\n",
+    "_ = evaluate_predictions(qeec_pred_job_2, y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e94784a-3599-4911-ade9-4792b50c31bb",
+   "metadata": {},
+   "source": [
+    "### Regularization\n",
+    "\n",
+    "In this configuration we keep `num_learners = 120` but use adaptive regularization to control sparsity more intelligently.\n",
+    "\n",
+    "- `regularization = \"auto\"` lets the optimizer automatically tune the penalty strength rather than fixing it to a constant value. It searches for a value that achieves a balanced subset of learners instead of overfitting to the training data.\n",
```

Comment:
```suggestion
    "- With `regularization = \"auto\"`, the optimizer automatically tunes the penalty strength rather than fixing it to a constant value. It searches for a value that achieves a balanced subset of learners instead of overfitting to the training data.\n",
```
(Trying to avoid "lets", "enables", etc.)